org.adempiere.base/src/org/compiere/wf/MWFActivity.java

```diff
@@ -990,7 +990,11 @@ else if (savepoint != null)
 				if (m_process != null)
 				{
 					m_process.setProcessMsg(this.getTextMsg());
-					m_process.saveEx();
+					try {

```

Comment:
this looks dangerous - it is ignoring exceptions when saving MWFProcess - is that what you want?